### ISSUE TITLE
Weird/corrupted fonts in EmulationStation

### ISSUE THREAD

**Issue by C-Fu** [RetroPie/RetroPie-Setup#896]
Running on the latest Raspbian + RetroPie beta 4

http://i.imgur.com/rsBZchF.jpg

Should also note that I did this before installing RetroPie to remove all traces of x11 + desktop:

$ sudo apt-get install deborphan
$ sudo apt-get autoremove --purge libx11-.* lxde-.* raspberrypi-artwork xkb-data omxplayer penguinspuzzle sgml-base xml-core alsa-.* cifs-.* samba-.* fonts-.* desktop-* gnome-.*
$ sudo apt-get autoremove --purge $(deborphan)
$ sudo apt-get autoremove --purge
$ sudo apt-get autoclean

**Comment by joolswills** [comment]
Unable to reproduce, but you are removing lots of stuff that retropie-setup will need and re-install - if you want a cut down raspbian why not start with the retropie image which has unneeded stuff already removed ?

**Comment by InsecureSpike** [comment]
i've notice this can happen, when switching from themes, if themes contain different font sets, so not so much as a corruption as maybe an unavoidable glitch, maybe

**Comment by amigagamer** [comment]
Ive had similar problems with display, especially fonts.  For me it was resolved by increasing my GPU memory.  Try setting 128 or 256mb to graphics and see if it resolves the issue.

**Comment by joolswills** [comment]
Good point.

**Comment by C-Fu** [comment]
Sorry for the late reply. 

"why not start with the retropie image which has unneeded stuff already removed "

[@joolswills]
I couldn't because I'm using berryboot, and berryboot's RetroPie 2.3 image doesn't seem to be working if I update it to current (that dreaded missing system error).

So I had to use the Raspbian image, remove desktop + x11, and install retropie manually.

But once I tried setting gpu_mem to 256 from 128, problem solved! :dancers: Cheers [@amigagamer]!

**Comment by joolswills** [comment]
closing as resolved.

**Comment by davorb** [comment]
This just happened to me. These are the things I did to get the issue:

1. ```dd``` the sd-card image to a card and boot up retropie
2. Put in a usb disk and let retropie crate the different folders
3. In the folder that lets you move settings to the pi, extract [the themes](http://blog.petrockblock.com/retropie/retropie-downloads/theme-archive-for-emulationstation/) to a folder called ```emulationstation/themes```. 
4. After I restarted I started experiencing the issues with the fonts.
